packages/grid/_modules_/grid/hooks/features/columnReorder/useColumnReorder.tsx

```diff
@@ -1,24 +1,24 @@
 import * as React from 'react';
-import { ColDef } from '../../models/colDef';
-import { useLogger } from '../utils';
-import { ApiRef } from '../../models';
+import { ColDef } from '../../../models/colDef';
+import { useLogger } from '../../utils';
```

Comment:
Full path here please. Cyclic dependency avoidance

Reply:
Could we drop the index.js files in this case?

Reply:
This is important at the hook level especially

They help the typings, and facilitate consumer imports of nested function I think

Reply:
I'm confused. Are you saying we need `../../utils/index.ts` but we can use it to important files?

Reply:
in the build we ref indexes, that cascade and build typings accordingly, then concat.
We can use it to import modules with it but we have to be cautious how we do it.
Components can import using indexes. But models for example, that are used everywhere, can create cyclic dependencies

Reply:
> in the build we ref indexes, that cascade and build typings accordingly, then concat.

Interesting, I was hoping the build could contact only using the imports used in the code.